### ISSUE TITLE
INJIVER-297: Inji verify: Observed access violation when "IBM Equal Access Accessibility Checker" executed on inji-verify  portal

### ISSUE THREAD

**Issue by Ivanmeneges** [mosip/inji-verify#1897]
1. Add extension "IBM Equal Access Accessibility Checker" extension to the browser

1. Right click on the browser and select inspect mode

1. Select Accessibility Assessment from the menu option

1. Click on scan button and observe the failures

Actual Result: there are accessibility violations observed while running the "IBM Equal Access Accessibility Checker" tool
Exected Result: There should not be any access violations on the pages

### Attachments
- [Accessibility_Report-Inji Verify_Home page.html](https://mosip.atlassian.net/browse/INJIVER-297)
- [Accessibility_Report-Inji Verify_Home page.xlsx](https://mosip.atlassian.net/browse/INJIVER-297)
- [Accessibility_Report-Inji Verify_Result page.html](https://mosip.atlassian.net/browse/INJIVER-297)
- [Accessibility_Report-Inji Verify_Result page.xlsx](https://mosip.atlassian.net/browse/INJIVER-297)
- [Accessibility_Report-Inji Verify_scan_page.html](https://mosip.atlassian.net/browse/INJIVER-297)
- [Accessibility_Report-Inji Verify_Scan_page.xlsx](https://mosip.atlassian.net/browse/INJIVER-297)
- [mosip-api-internal.qa-inji-mimoto-1716541541751-report_T-157_P-107_S-1_F-49.html](https://mosip.atlassian.net/browse/INJIVER-297)

**Comment by Ivanmeneges** [comment]
sanchi singh commented: Check on details with [~accountid:639195c95fc160544e1df6df]
